src/components/Events/EventDetail/EventDetail.tsx

```diff
@@ -1,24 +1,40 @@
+import ClayLoadingIndicator from '@clayui/loading-indicator'
 import { useQuery } from '@tanstack/react-query'
 import fetchEvent from 'api/fetch-event'
+import { ClayLoadingIndicatorWrapperStyled } from 'assets/styles/containers'
 const EventDetail = ({ eventId }: IProps) => {
-  const eventDetail = useQuery(['event'], () => fetchEvent({ id: eventId }))
-  const targetString: string =
-    'Podréis asistir en directo por streaming en [youtube](https://www.youtube.com/watch?v=8BGfDJ1j5-Q).\n\nEn la [página del evento ](https://lugspain.github.io/mesdeqa/) podéis consultar más información sobre las sesiones, las comunidades y los ponentes que participan.'
-  const rExp: RegExp = /https:\/\/www\.youtube\.com\/watch\?v=.{11}/
-  const result: RegExpMatchArray | null = targetString.match(rExp)
+  const { status, data } = useQuery(['event'], () =>
+    fetchEvent({ id: eventId })
+  )
+
+  if (data) {
+    const rExp: RegExp = /https:\/\/www\.youtube\.com\/watch\?v=.{11}/
+    const result: RegExpMatchArray | null = data.description.match(rExp)
+    var resultString: string | undefined = result?.pop()
+    var replace = /watch\?v=/gi
```

Comment:
This can change over the meetups, not sure if it will work in all cases...
Have you seen this SO answers?
https://stackoverflow.com/questions/3452546/how-do-i-get-the-youtube-video-id-from-a-url